src/Features/Core/Portable/Completion/CompletionOptions.cs

```diff
@@ -169,18 +172,13 @@ public Metadata()
                 new(nameof(CompletionOptions), nameof(TriggerInArgumentLists), defaultValue: true,
                 storageLocation: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.TriggerInArgumentLists"));
 
-            /// <summary>
-            /// Indicates if the completion is trigger by toggle the expander.
-            /// </summary>
-            public static readonly Option2<bool> IsExpandedCompletion
-                = new("CompletionServiceOptions", nameof(IsExpandedCompletion), defaultValue: false);
-
-            /// <summary>
-            /// Timeout value used for time-boxing completion of unimported extension methods.
-            /// Value less than 0 means no timebox; value == 0 means immediate timeout (for testing purpose)
-            /// </summary>
-            public static readonly Option2<int> TimeoutInMillisecondsForExtensionMethodImportCompletion
-                = new("CompletionServiceOptions", nameof(TimeoutInMillisecondsForExtensionMethodImportCompletion), defaultValue: 500);
+            // Test-only option
+            public static readonly Option2<bool> ForceExpandedCompletionIndexCreation
+                = new(nameof(CompletionOptions), nameof(ForceExpandedCompletionIndexCreation), defaultValue: false);
+
+            // Test-only option
+            public static readonly Option2<bool> BlockOnExpandedCompletion
+                = new(nameof(CompletionOptions), nameof(BlockOnExpandedCompletion), defaultValue: false);
```

Comment:
can you explain what the test options do?

Reply:
This one is actually not used by changes in this PR, it is intended for the change in the follow up https://github.com/dotnet/roslyn/pull/59045

I will remove this from this PR then